securedrop/tests/test_i18n.py

```diff
@@ -211,6 +219,29 @@ def test_verify_default_locale_en_us_if_not_defined_in_config(self):
         finally:
             config.DEFAULT_LOCALE = DEFAULT_LOCALE
 
+    def test_locale_to_rfc_5646(self):
+        assert i18n.locale_to_rfc_5646('en') == 'en'
+        assert i18n.locale_to_rfc_5646('en-US') == 'en'
+        assert i18n.locale_to_rfc_5646('en_US') == 'en'
+        assert i18n.locale_to_rfc_5646('en-us') == 'en'
+        assert i18n.locale_to_rfc_5646('zh-hant') == 'zh-Hant'
+
+    def test_html_lang_correct(self):
+        app = journalist.app.test_client()
+        resp = app.get('/', follow_redirects=True)
+        html = resp.data.decode('utf-8')
+        assert re.compile('<html .*lang="en".*>').search(html), html
+
+        app = source.app.test_client()
+        resp = app.get('/', follow_redirects=True)
+        html = resp.data.decode('utf-8')
+        assert re.compile('<html .*lang="en".*>').search(html), html
+
+        # check '/generate' too because '/' uses a different template
```

Comment:
I don't see how this is different from the above ? did you mean app.get('/generate') ?